src/bpftrace.cpp

```diff
@@ -357,54 +355,87 @@ std::vector<uint64_t> BPFtrace::get_arg_values(std::vector<Field> args, uint8_t*
         switch (arg.type.size)
         {
           case 8:
-            arg_values.push_back(*(uint64_t*)(arg_data+arg.offset));
+            arg_values.push_back(
+              std::make_unique<PrintableInt>(
+                *(uint64_t*)(arg_data+arg.offset)));
```

Comment:
Should probably reinterpret cast here instead of C-style cast. Same for all the other cases
```suggestion
                *reinterpret_cast<uint64_t*>(arg_data+arg.offset)));
```